frontend/packages/container-security/src/components/ImageVulnerabilityTogggleGroup.scss

```diff
@@ -0,0 +1,24 @@
+.cs-imagevulnerability-details {
+  display: flex;
+  &__donut {
+    padding: 30px;
+    width: 20%;
+  }
+  &__summary {
+    width: 80%;
+  }
+  &__summary-list {
+    display: flex;
+    flex-direction: column;
+    &--title {
+      margin: 5px;
+    }
+  }
+  &__empty-text {
+    display: flex;
+    min-height: 294px;
+    max-width: 359px;
```

Comment:
Why are we using this specific number for min height and min width

Reply:
We need this to match this design, where we need an empty state message should be aligned vertically in the middle.

![image](https://user-images.githubusercontent.com/2561818/116707912-d1132180-a9ec-11eb-8919-ad556b75d296.png)